alaoweb/2021.alaoweb.org
# added keyword/audience to session displays / search

Closes #19 

On the schedule and schedule-search pages, the session modals now include IGs and keywords.

The IGs are now listed as an array of acronyms in sessions.yml, and there is a _data/acronyms.yml file to convert them from acronyms to text

The session search page is now searchable by keywords and IG name (e.g. "PROMIG" or "programming..." 

To test: 
* on the schedule page, open a regular content session and see the "keywords" and "audience" fields
* on the schedule-search, try searching by text "Promig" or "programming" to find relevant content
* you should not see the keyword and audience fields on meetings, keynotes, and the like
